pkg/es/config/config_test.go

```diff
@@ -0,0 +1,740 @@
+// Copyright (c) 2024 The Jaeger Authors.
+//
```

Comment:
incorrect header. Delete and run `make fmt`

Reply:
okay done now,
sorry prettier generated the previous one